certified-connectors/Document Merge/Document-Merge.swagger.json

```diff
@@ -0,0 +1,172 @@
+            "description": "Available when the Document and Mail Merge Add-In is installed - www.documentmerge.com",
+            "x-ms-summary": "Pre-Config Template"
+          },
+          {
+            "name": "Source",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "description": "Set the path to the source document Template e.g. \"LibraryName\\TemaplateName.docx\"",
+            "x-ms-summary": "Source"
+          },
+          {
+            "name": "Destination",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "description": "The Library name to store generated document",
+            "x-ms-summary": "Destination"
+          },
+          {
+            "name": "SaveAsPDF",
+            "in": "query",
+            "required": false,
+            "type": "boolean",
+            "description": "Set to save an accompany PDF file",
+            "x-ms-summary": "Save as PDF"
+          },
+          {
+            "name": "SaveAsPDFOnly",
+            "in": "query",
+            "required": false,
+            "type": "boolean",
+            "description": "Set to save only a PDF file no accompanying document",
+            "x-ms-summary": "Save as PDF only"
+          },
+          {
+            "name": "SaveAsPDFA",
+            "in": "query",
+            "required": false,
+            "type": "boolean",
+            "description": "Set to save PDF file as PDF\\A format",
+            "x-ms-summary": "Save as PDF/A"
+          },
+          {
+            "name": "DisplayImage",
+            "in": "query",
+            "required": false,
+            "type": "boolean",
+            "description": "Will display Image instead of hyperlink of image",
+            "x-ms-summary": "Display Image"
+          },
+          {
+            "name": "OutputFileName",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "description": "The name to save the file - can include list internal column names - see manual www.documentmerge.com",
+            "x-ms-summary": "Output file name"
+          },
+          {
+            "name": "OverWrite",
+            "in": "query",
+            "required": false,
+            "type": "boolean",
+            "description": "Overwrite the document if same name",
+            "x-ms-summary": "Overwrite"
+          },
+          {
+            "name": "SendMail",
+            "in": "query",
+            "required": false,
+            "type": "boolean",
+            "description": "Available when the Document and Mail Merge Add-In is installed",
+            "x-ms-summary": "Send mail"
+          },
+          {
+            "name": "MailTemplate",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "description": "Specify if SendMail Option selected e.g. \"TemplateTitleName\"",
```

Comment:
the word "SendMail" will not be translated into other languages. if your connector meets our [requirements](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-to-ga#required-criteria) for [General Availability](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-to-ga) it will become eligible for translation into other languages. id that okay with you? if yes please resolve this conversation.